### PR TITLE
Feature/ui update1.0.22

### PR DESCRIPTION
**Pull request by alexxcheung**

### CODE REVIEW

File: LetsDinnerV2 MessagesExtension/Controller/EventSummaryVC/EventSummaryViewController.swift [hunk 1]
```diff
@@ -305,7 +299,7 @@ extension EventSummaryViewController: UITableViewDelegate, UITableViewDataSource
             }
         }
         
-        // Netural - Pending
+        // MARK: Netural - Pending
```
Comment (Ericordo) [inline LetsDinnerV2 MessagesExtension/Controller/EventSummaryVC/EventSummaryViewController.swift]: Typo

File: LetsDinnerV2 MessagesExtension/Controller/NewEventVC/NewEventViewController.swift [hunk 2]
```diff
@@ -166,7 +166,10 @@ class NewEventViewController: UIViewController  {
     }
     
      private func allFieldsAreFilled() -> Bool {
-        guard let host = hostNameTextField.text, let dinner = dinnerNameTextField.text, let location = locationTextField.text, let date = dateTextField.text else {
+        guard let host = hostNameTextField.text,
```
Comment (Ericordo) [inline LetsDinnerV2 MessagesExtension/Controller/NewEventVC/NewEventViewController.swift:169]: Does this prevent an empty string to be validated?

File: LetsDinnerV2 MessagesExtension/Controller/SelectedRecipesVC/SelectedRecipesViewController.swift [hunk 3]
```diff
@@ -61,7 +61,8 @@ class SelectedRecipesViewController: UIViewController {
             UIView.animate(withDuration: 0.7,
                            delay: 0.0, options: .transitionCrossDissolve,
                            animations: { self.rearrangeTextLabel.alpha = 0.1 },
-                           completion: { finished in self.rearrangeTextLabel.isHidden = true})
+                           completion: { finished in
+                            self.rearrangeTextLabel.isHidden = true})
```
Comment (Ericordo) [inline LetsDinnerV2 MessagesExtension/Controller/SelectedRecipesVC/SelectedRecipesViewController.swift:65]: if finished is not used, replace with _

File: LetsDinnerV2 MessagesExtension/Controller/SelectedRecipesVC/SelectedRecipesViewController.swift [hunk 4]
```diff
@@ -256,39 +258,39 @@ extension SelectedRecipesViewController: UITableViewDelegate, UITableViewDataSou
                 if let index = Event.shared.selectedCustomRecipes.firstIndex(where: { $0.id == cell.selectedCustomRecipe.id }) {
                     
                     Event.shared.reassignCustomOrderAfterRemoval(recipeType: .customRecipes, index: index)
-                    Event.shared.selectedCustomRecipes.remove(at: index)
-                    self.updateLocalVariable()
-                    self.updateNumberOfTotalRecipe()
```
Comment (Ericordo) [inline LetsDinnerV2 MessagesExtension/Controller/SelectedRecipesVC/SelectedRecipesViewController.swift:261]: totalNumberOfRecipes

File: LetsDinnerV2 MessagesExtension/Controller/SelectedRecipesVC/SelectedRecipesViewController.swift [hunk 5]
```diff
@@ -256,39 +258,39 @@ extension SelectedRecipesViewController: UITableViewDelegate, UITableViewDataSou
...
+            
+            UIView.animate(withDuration: 0.3,
+                           delay: 0.0,
+                           options: .transitionCrossDissolve,
+                           animations: {
+                            cell.alpha = 0.3
+                            cell.heightConstraint.constant = 0
+                            cell.backgroundCellView.layoutIfNeeded() },
+                           completion: { finished in
+                            self.recipesTableView.reloadData()
```
Comment (Ericordo) [inline LetsDinnerV2 MessagesExtension/Controller/SelectedRecipesVC/SelectedRecipesViewController.swift:287]: finished not used, replace with _

File: LetsDinnerV2 MessagesExtension/Controller/TasksListVC/TasksListViewController.swift [hunk 6]
```diff
@@ -77,19 +77,35 @@ class TasksListViewController: UIViewController {
     private func setupUpdateButton() {
         submitButton.layer.masksToBounds = true
         submitButton.layer.cornerRadius = 12
-        submitButton.setGradient(colorOne: Colors.newGradientPink, colorTwo: Colors.newGradientRed)
-        submitButton.alpha = 0.5
```
Comment (Ericordo) [inline LetsDinnerV2 MessagesExtension/Controller/TasksListVC/TasksListViewController.swift:81]: Why changing the alpha?

File: LetsDinnerV2 MessagesExtension/Manager/TestManager.swift [hunk 7]
```diff
@@ -128,8 +134,8 @@ class TestManager {
     static func quickFillIn() -> Event {
         let event = Event()
         
-        event.dinnerName = "Quick Dinner (Finish in 10 mins)"
-        event.hostName = "Obama"
+        event.dinnerName = "Quick Dinner (tesing)"
```
Comment (Ericordo) [inline LetsDinnerV2 MessagesExtension/Manager/TestManager.swift]: typo

File: LetsDinnerV2 MessagesExtension/Controller/ManagementVC/ManagementViewController.swift [hunk 8]
```diff
@@ -330,6 +337,15 @@ class ManagementViewController: UIViewController {
         }
     }
     
+    private func hideFooterView() {
+        guard let footerView = footerView else { return }
+        if tasks.isEmpty {
+            footerView.isHidden = true
+        } else {
+            footerView.isHidden = false
+        }
```
Comment (Ericordo) [inline LetsDinnerV2 MessagesExtension/Controller/ManagementVC/ManagementViewController.swift]: footerView.isHidden = tasks.isEmpty

File: LetsDinnerV2 MessagesExtension/Controller/NewEventVC/NewEventViewController.swift (hunk 2 above)
Comment (alexxcheung) [reply LetsDinnerV2 MessagesExtension/Controller/NewEventVC/NewEventViewController.swift:169]: its used for safely unwrapped the optional

Comment (alexxcheung) [reply LetsDinnerV2 MessagesExtension/Controller/NewEventVC/NewEventViewController.swift:169]: The code below is for the validation

**Comment by alexxcheung** [comment]
Updated

**Review by Ericordo — APPROVED** [review]

**Comment by alexxcheung** [comment]
Are you going to merge the branch?

**Comment by Ericordo** [comment]
I approved, you merge

**Comment by Ericordo** [comment]
And you have to resolve conflicts

**Pull request merged**